x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/model/rule_assets/prebuilt_rule_asset.test.ts

```diff
@@ -7,8 +7,19 @@
 
 import { expectParseError, expectParseSuccess, stringifyZodError } from '@kbn/zod-helpers';
 import { getListArrayMock } from '../../../../../../common/detection_engine/schemas/types/lists.mock';
-import { PrebuiltRuleAsset } from './prebuilt_rule_asset';
+import { PrebuiltRuleAsset, TypeSpecificFields } from './prebuilt_rule_asset';
 import { getPrebuiltRuleMock, getPrebuiltThreatMatchRuleMock } from './prebuilt_rule_asset.mock';
+import { TypeSpecificCreateProps } from '@kbn/security-solution-plugin/common/api/detection_engine';
+
+describe('TypeSpecificFields', () => {
```

Comment:
Why does it have its own top-level describe block? Isn't it in fact a part of `'Prebuilt rule asset schema'`?

Reply:
Moved within `Prebuilt rule asset schema`